src/EnergyPlus/Material.hh

```diff
@@ -132,232 +132,224 @@ namespace Material {
+        Array2D<Real64> GasCon = Array2D<Real64>(3, 5, 0.0);        // Gas conductance coefficients for up to 5 gases in a mixture
+        Array2D<Real64> GasVis = Array2D<Real64>(3, 5, 0.0);        // Gas viscosity coefficients for up to 5 gases in a mixture
+        Array2D<Real64> GasCp = Array2D<Real64>(3, 5, 0.0);         // Gas specific-heat coefficients for up to 5 gases in a mixture
+        Array1D<Real64> GasWght = Array1D<Real64>(5, 0.0);          // Gas molecular weight for up to 5 gases in a mixture
+        Array1D<Real64> GasSpecHeatRatio = Array1D<Real64>(5, 0.0); // Gas specific heat ratio (used for low pressure calculations)
+        Array1D<Real64> GasFract = Array1D<Real64>(5, 0.0);         // Gas fractions for up to 5 gases in a mixture
         // Radiation parameters
-        Real64 AbsorpSolar;              // Layer solar absorptance
-        Real64 AbsorpSolarInput;         // Layer solar absorptance input by user
-        bool AbsorpSolarEMSOverrideOn;   // if true, then EMS calling to override value for solar absorptance
-        Real64 AbsorpSolarEMSOverride;   // value to use when EMS calling to override value for solar absorptance
-        Real64 AbsorpThermal;            // Layer thermal absorptance
-        Real64 AbsorpThermalInput;       // Layer thermal absorptance input by user
-        bool AbsorpThermalEMSOverrideOn; // if true, then EMS calling to override value for thermal absorptance
-        Real64 AbsorpThermalEMSOverride; // value to use when EMS calling to override value for thermal absorptance
-        Real64 AbsorpVisible;            // Layer Visible Absorptance
-        Real64 AbsorpVisibleInput;       // Layer Visible Absorptance input by user
-        bool AbsorpVisibleEMSOverrideOn; // if true, then EMS calling to override value for visible absorptance
-        Real64 AbsorpVisibleEMSOverride; // value to use when EMS calling to override value for visible absorptance
+        Real64 AbsorpSolar = 0.0;                // Layer solar absorptance
+        Real64 AbsorpSolarInput = 0.0;           // Layer solar absorptance input by user
+        bool AbsorpSolarEMSOverrideOn = false;   // if true, then EMS calling to override value for solar absorptance
+        Real64 AbsorpSolarEMSOverride = false;   // value to use when EMS calling to override value for solar absorptance
+        Real64 AbsorpThermal = 0.0;              // Layer thermal absorptance
+        Real64 AbsorpThermalInput = 0.0;         // Layer thermal absorptance input by user
+        bool AbsorpThermalEMSOverrideOn = false; // if true, then EMS calling to override value for thermal absorptance
+        Real64 AbsorpThermalEMSOverride = 0.0;   // value to use when EMS calling to override value for thermal absorptance
+        // dynamic thermal and solar absorptance coating parameters
+        VariableAbsCtrlSignal variableAbsorptanceCtrlSignal = VariableAbsCtrlSignal::Invalid;
```

Comment:
Ok, so all of the variables here are called `AbsorpSolar` and `AbsorpThermal`.  So maybe rename new variables to match, e.g., `absorpVarCtrlSignal`, `absorpSolarVarSchedIdx`, `absorpThermalVarFuncIdx`, etc.

Reply:
renamed them. look more consistent this way